projects/Grandine-Support-Documentation.md

```diff
@@ -0,0 +1,79 @@
+# Grandine Support and Documentation
+
+This project is focused on building out a more comprehensive support and documentation book for Grandine through research and trial and error. 
+
+## Motivation
+
+Grandine is the newest Consensus Layer client and the documentation is on the lighter side. I think it's important to have more comprehsneive documentation around Grandine and its capabilities. I also think this will help attract new users to the client, which would increase client diversity among CL clients.
+
+## Project description
+
+I am proposing to add more support and documentation to [Grandine's book](https://github.com/grandinetech/grandine/tree/develop/book) which will require me to become intimately familiar with Grandine through research and trial and error. 
+
+The following areas are ones I think should be addressed but this is likely to change as I begin making my way deeper into Grandine. I am taking inspiration from the Lighthouse and Prysm documentation books, so whilst not everything that they have will be applicable to Grandine, I think it is a good base to work from.
+- ### Installation
+- ### Eth-docker integration
+- ### Validator Client
+- ### Contributing/Testing
+
+## Specification
+
+I have already begun to make myself familiar with Grandine and the CL in general. The Grandine Discord channel is becoming active for queries around errors from other fellows as well. This will be a good method and way to keep track of issues others are facing and it also fits in nicely with everybody working more collaboratively together.
+
+I believe that these solutions can be implemented through research and trial and error. Since this is all new to me, I need to personally perform these tasks so that I can learn and also understand what pain points there are for new users. For more technical topics that other fellows and users are working on, I will have to rely on them to relay to me where there are errors in the current documentation and what solutions they found to get through their blockers.  
+
+## Roadmap
+
+**By end of July:**
+
+   - Have Installation section complete, including support for non Linux systems
+
+**By end of August:**
+
+   - Have Eth-Docker integration section complete
+
+**By end of September:**
+
+   - Have Validator Client section complete
+
+**By end of October:**
+
+   - Have Contributing/Testing section complete
+
+**By Devcon VII:**
+
+   - Complete other sections that were identified for updates
+
+## Possible challenges
+
+I am still learning how to code (focusing on Rust) and building up my technical chops by learning more about the Consensus Layer and the Ethereum protocol in general. Working on this documentation for Grandine, while not as technical as other projects, will still require a great deal of technical understanding through trial and error. 
```

Comment:
I think this might be helpful for dev focused docs. You can document what gives you trouble when diving into grandine and explain how their codebase works for contributor docs

Reply:
@sauliusgrigaitis cheers! Yes I came across that this weekend when researching too! I noticed there weren't any for Grandine. 

@taxmeifyoucan agreed! That was my thinking behind doing this.